x-pack/plugins/security_solution/public/common/components/navigation/use_security_solution_navigation/use_navigation_items.tsx

```diff
@@ -133,6 +133,7 @@ function usePrimaryNavigationItemsToDisplay(navTabs: Record<string, NavTab>) {
             {
               ...securityNavGroup[SecurityNavGroupKey.manage],
               items: [
+                // TODO: also hide other management pages based on authz privileges
```

Comment:
This is great! but could you add a ticket number next to the TODO so we have a reference when we come upon it.

Reply:
Thank you for the suggestion! We got private tickets for it, so instead of referencing them in the code, I mentioned this PR and this code part in the epic, so we won't forget about this.